packages/components/src/Avatar/src/Avatar.module.css

```diff
@@ -0,0 +1,161 @@
+.hop-Avatar  {
+    /* Default */
+    --hop-Avatar-border-radius: var(--hop-shape-circle);
+    --hop-Avatar-mix-blend-mode: normal;
+
+    /* Colors */
+    --hop-Avatar-decorative-1-background-color: var(--hop-decorative-option1-surface-strong);
+    --hop-Avatar-decorative-2-background-color: var(--hop-decorative-option2-surface-strong);
+    --hop-Avatar-decorative-3-background-color: var(--hop-decorative-option3-surface-strong);
+    --hop-Avatar-decorative-4-background-color: var(--hop-decorative-option4-surface-strong);
+    --hop-Avatar-decorative-5-background-color: var(--hop-decorative-option5-surface-strong);
+    --hop-Avatar-decorative-6-background-color: var(--hop-decorative-option6-surface-strong);
+    --hop-Avatar-decorative-7-background-color: var(--hop-decorative-option7-surface-strong);
+    --hop-Avatar-decorative-8-background-color: var(--hop-decorative-option8-surface-strong);
+    --hop-Avatar-decorative-9-background-color: var(--hop-decorative-option9-surface-strong);
+    --hop-Avatar-decorative-1-text-color: var(--hop-decorative-option1-text);
+    --hop-Avatar-decorative-2-text-color: var(--hop-decorative-option2-text);
+    --hop-Avatar-decorative-3-text-color: var(--hop-decorative-option3-text);
+    --hop-Avatar-decorative-4-text-color: var(--hop-decorative-option4-text);
+    --hop-Avatar-decorative-5-text-color: var(--hop-decorative-option5-text);
+    --hop-Avatar-decorative-6-text-color: var(--hop-decorative-option6-text);
+    --hop-Avatar-decorative-7-text-color: var(--hop-decorative-option7-text);
+    --hop-Avatar-decorative-8-text-color: var(--hop-decorative-option8-text);
+    --hop-Avatar-decorative-9-text-color: var(--hop-decorative-option9-text);
+
+    /* xs */
```

Comment:
wouldn't you want to add the comment Sizes, and add all the variable under. the comment sm/md/lg seems redundant

Reply:
I could, but splitting it by size makes it clearer and ensures that you won't mix the props up.

Reply:
I guess there's only one variable per size, I'll do this.